stellarphot/visualization/seeing_profile_functions.py

```diff
@@ -317,13 +293,15 @@ def __init__(self, data, x, y):
 
 
         """
-        self.cen = find_center(data, (x, y), cutout_size=30)
-        self.data = data
+        self._cen = find_center(data, (x, y), cutout_size=30)
+        self._data = data

```

Comment:
After a quick check it looks like this change is fine -- no code modifies these attributes (I think)